pkg/registryserver/registryserver.go

```diff
@@ -13,12 +12,28 @@ import (
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 	"k8s.io/klog/glog"
+	"k8s.io/utils/keymutex"
 )
 
+// RegistryListener is an registry server listener interface
+// All the callbacks are called once after updating the in-memory
+// registry data.
+type RegistryListener interface {
+	// OnNodeAdded is called by RegistryServer whenever a new node controller is registered
+	// or node controller updated its endpoint.
+	OnNodeAdded(ctx context.Context, node *NodeInfo)
+	// OnNodeDeleted is called by RegistryServer whenever a node controller unregistered.
+	// Callback implementations has to note that by the time this method is called,
+	// the NodeInfo for that node have already removed from in-memroy registry.
```

Comment:
That's redundant, the same is said above for RegistryListener interface. There's also a typo (`in-memroy`). Do you want to fix this or get it merged as-is?

Reply:
Fixed, typo.